instrumentation/micrometer/micrometer-1.5/library/src/main/java/io/opentelemetry/instrumentation/micrometer1shim/Bridging.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.micrometer1shim;
```

Comment:
This package name reflects the pattern in [opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java/blob/main/micrometer1-shim/src/main/java/io/opentelemetry/micrometer1shim/Bridging.java#L6), but is different than the original `io.opentelemetry.instrumentation.micrometer.v1_5`. 

Happy to change it back to `io.opentelemetry.instrumentation.micrometer.v1_5`.

Reply:
I think I'd prefer `io.opentelemetry.instrumentation.micrometer.v1_5`, since it follows the conventions of this repo (and we're changing the package name anyway)